src/Avalonia.Themes.Fluent/Controls/ColorSpectrum.xaml

```diff
@@ -0,0 +1,136 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:CompileBindings="True"
+        xmlns:converters="using:Avalonia.Controls.Converters">
+
+  <Design.PreviewWith>
+    <Border Padding="20">
+      <ColorSpectrum Width="256" Height="256" />
+    </Border>
+  </Design.PreviewWith>
+
+  <Styles.Resources>
+    <converters:EnumValueEqualsConverter x:Key="EnumValueEquals" />
+  </Styles.Resources>
+
+  <Style Selector="ColorSpectrum">
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate>
+          <Grid x:Name="PART_LayoutRoot">
+            <Grid x:Name="PART_SizingGrid"
+                  HorizontalAlignment="Center"
+                  VerticalAlignment="Center"
+                  ClipToBounds="True">
```

Comment:
I also just noticed SizingGrid sits directly inside LayoutRoot... why UWP did this I don't know and it's completely unnecessary it seems.

I'm going to combine both of these and change it to Panel. That should improve performance and simplify things.